tests/orion/api/test_block_schemas.py

```diff
@@ -128,22 +128,22 @@ async def test_create_block_schema(self, session, client, block_type_x):
     async def test_create_block_schema_with_existing_checksum_fails(
         self, session, client, block_type_x
     ):
-        response = await client.post(
+        response_1 = await client.post(
             "/block_schemas/",
             json=BlockSchemaCreate(fields={}, block_type_id=block_type_x.id).dict(
                 json_compatible=True
             ),
         )
-        assert response.status_code == 201
+        assert response_1.status_code == 201
 
-        response = await client.post(
+        response_2 = await client.post(
             "/block_schemas/",
             json=BlockSchemaCreate(fields={}, block_type_id=block_type_x.id).dict(
                 json_compatible=True
             ),
         )
-        assert response.status_code == 409
-        assert "Identical block schema already exists." in response.json()["detail"]
+        assert response_2.status_code == 201
```

Comment:
```suggestion
        assert response_2.status_code == 200
```
nit: we should be returning 200 if the object already exists

Reply:
Updated in https://github.com/PrefectHQ/prefect/pull/6301/commits/4dabf22c042f4bb136e46a323c7654b2125955c1